types/actions/ommail.pp

```diff
@@ -0,0 +1,11 @@
+# Struct data type for the Rsyslog E-Mail module's Action options
+type Rsyslog::Actions::Ommail = Struct[{
+  Server             => Variant[Stdlib::Fqdn, Stdlib::IP::Address],
+  Port               => Stdlib::Port,
+  MailFrom           => Pattern[/.+@.+\..+/],
+  MailTo             => Pattern[/.+@.+\..+/],
```

Comment:
@alexjfisher @ekohl @bastelfreak @binford2k - thoughts on if this is the best regex for a naive email validation or should it be simpler/more specific?

Reply:
TBH, that'll probably do.  See https://emailregex.com/ and https://www.regular-expressions.info/email.html

Reply:
Do note that your current regex doesn't use start (`^`) and end (`$`) symbols so it's equivalent to `/@.+\./`